game-app/game-headed/build.gradle

```diff
@@ -5,6 +5,14 @@ plugins {
     id "de.undercouch.download" version "5.6.0"
 }
 
+//Activate the locked apple API with flag '--add-opens=java.desktop/com.apple.eawt.event=ALL-UNNAMED'
+application {
+    mainClass = 'org.triplea.game.client.HeadedGameRunner'
+    applicationDefaultJvmArgs = [
+            '--add-opens=java.desktop/com.apple.eawt.event=ALL-UNNAMED'
```

Comment:
minor: please use double quotes instead of single quotes

Reasoning: we would like to move to Kotlin instead of Groovy for the gradle build file. Kotlin only allows double quotes, so we are closer to kotlin syntax with double quotes.